USER_MANUAL.md

```diff
@@ -124,6 +124,11 @@ Email: <sally@highwoodemissions.com>
       - [\<redundancy\_filter\>](#redundancy_filter)
       - [\<sort\_by\_rate\>](#sort_by_rate)
       - [\<threshold\>](#threshold)
+    - [rolling\_average](#rolling_average)
```

Comment:
See my comments about separating continuous and mobile in implementation - they apply here too